### PR TITLE
add ch.are.wohnungsinventar-zweitwohnungsanteil

### PR DESCRIPTION
**Pull request by ltclm**
[Testlink](https://mf-geoadmin3.int.bgdi.ch/?topic=ech&lang=de&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.are.wohnungsinventar-zweitwohnungsanteil&api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fltclm&X=190000.00&Y=660000.00&zoom=1&wms_url=%2F%2Fwms-bgdi.dev.bgdi.ch%2F)

### CODE REVIEW

File: chsdi/models/vector/are.py [hunk 1]
```diff
@@ -170,3 +170,25 @@ class Gemeindetypen_2012(Base, Vector):
...
+    __table_args__ = ({'schema': 'raumplanung', 'autoload': False})
+    __template__ = 'templates/htmlpopup/zweitwohnungsanteil.mako'
+    __bodId__ = 'ch.are.wohnungsinventar-zweitwohnungsanteil'
+    __label__ = 'id'
+    id = Column('objectid', Integer, primary_key=True)
+    zwg_3150 = Column('zwg_3150', Integer)
+    zwg_3010 = Column('zwg_3010', Integer)
+    zwg_3100 = Column('zwg_3100', Integer)
+    zwg_3110 = Column('zwg_3110', Numeric)
+    zwg_3120 = Column('zwg_3120', Numeric)
```
Comment (loicgasser) [inline chsdi/models/vector/are.py]: We don't use numeric anymore, please use float instead

File: chsdi/models/vector/are.py [hunk 2]
```diff
@@ -170,3 +170,25 @@ class Gemeindetypen_2012(Base, Vector):
...
+    zwg_3200_de = Column('zwg_3200_de', Text)
+    zwg_3200_fr = Column('zwg_3200_fr', Text)
+    zwg_3200_it = Column('zwg_3200_it', Text)
+    zwg_3200_rm = Column('zwg_3200_rm', Text)
+    zwg_3200_en = Column('zwg_3200_en', Text)
```
Comment (loicgasser) [inline chsdi/models/vector/are.py]: Unicode

File: chsdi/templates/htmlpopup/zweitwohnungsanteil.mako [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c,lang)">
+<%
+    lang = lang if lang in ('fr','it','rm','en') else 'de'
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/zweitwohnungsanteil.mako]: I don't think this does much, it already defaults to 'de' at a higher level in the app if the lang is not the available langs.

**Comment by loicgasser** [comment]
minor comments otherwise it's good to go.

**Comment by AFoletti** [comment]
In the tooltip, please round zwg_3110 and zwg_3120 to 1 decimal. Otherwise LGTM too, but please wait for Datenherr feedback before merge

**Comment by ltclm** [comment]
@loicgasser @AFoletti Thanks for your feedback, i have addressed your remarks.

**Comment by AFoletti** [comment]
thanks! Could you please add Gemeinde Name in tooltip too (first position)?

**Pull request merged**